Resources/Prototypes/Floof/Entities/Objects/Weapons/Melee.yml

```
@@ -91,3 +91,95 @@
   - type: BatterySelfRecharger
     autoRecharge: true
     autoRechargeRate: 40
+
+- type: entity
+  name: Halberd
+  parent: UnholyHalberd
+  id: Halberd
+  description: A polarm with an axe blade topped with a spike mounted on a long shaft. Produced by SESWC!
+  components:
+  - type: MeleeRequiresWield
+  - type: MeleeWeapon
+    heavyStaminaCost: 10
+  - type: Clothing
+    sprite: Floof/Objects/Weapons/Melee/halberd.rsi
+    quickEquip: false
+    slots:
+    - back
+  - type: Sprite
+    sprite: Floof/Objects/Weapons/Melee/halberd.rsi
+    state: icon
+
+- type: entity
+  name: Energy Halberd
+  parent: EnergySwordDouble
+  id: EHalberd
+  description: A polarm handle with an engery based axe head to it. Produced by SESWC!
+  components:
```

Prototype naming and grammar - will require changing other references
```suggestion
- type: entity
  name: energy halberd
  parent: WeaponEnergySwordDouble
  id: WeaponEnergyHalberd
  description: A polarm handle with an energy-based axe head to it. Produced by SESWC!
  components:
```

---

Im reverting the parent back as Upstream files is Just called "EnergySwordDouble"

---

That part's on me